README.md

```diff
@@ -83,21 +83,37 @@ Utilize the following steps to deploy the project
 1. Clone the repository
 
     ```
-    git clone https://github.com/matallen/community-ninja-board
+    git clone https://github.com/redhat-cop/ninja-board
     ```
 
 2. Change into the project directory and utilize Ansible Galaxy to retrieve required dependencies
 
     ```
     ansible-galaxy install -r requirements.yml --roles-path=galaxy
-    ``` 
- 
+    ```
+
 3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
 
-4. Execute the _openshift-applier_
+4. Update ansible variables in *.applier/inventory/group_vars/all.yml* to configure desired end result.
+
+   1. Update variable `ninja_board_dev_namespace` to some unique value such as `"{{ namespace }}-dev-<your-keberos-id>"`. This will be the name of the openshift project.
+   2. Update variable `access_namespace` to the namespace defined in 4.1.
+   3. **Optional:** If you want to deploy a different github repository (ie, a forked copy of redhat-cop/ninja-board), then update the ansible variable `source_repo`.
+
+5. Add the namespace defined in 4.1 above to the Openshift Jenkins Sync plugin
+
+   1. Login into the central Jenkins instance https://jenkins-ninja-board-cicd-dev.int.open.paas.redhat.com
```

Comment:
Can we remove the URL as it may not be representative of their environment

Reply:
@sabre1041 These instructions are specific to the openpaas environment. It would be great to make these more generic to work in all environments, but if that's the case, it won't work in openpaas due to various restrictions. Jenkins instance requires at least 1Gi to start and the jenkins-nodejs-agent takes about ~750Mi to build the front-end app taking up most of the 2Gi memory limit. A user also can't create more than one project in openpaas.